### PR TITLE
Dashboard for EmployeeManagers (backend)

### PR DESCRIPTION
**Pull request by Gkolocsar**
Dashboard and controller method for the EmployeeManagers.

Can you please review? @andresmoschini @AlphaGit @sofipacifico @joseguti84

### CODE REVIEW

**Comment by andresmoschini** [comment]
Looks fine

File: source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluatedEmployees.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        public override List<EmployeeEvaluationDTO> ExecuteWithResult()
+        {
+            RavenQueryStatistics stats;
+            IQueryable<EmployeeToEvaluate_Search.Projection> query = RavenSession
+                .Query<EmployeeToEvaluate_Search.Projection, EmployeeToEvaluate_Search>()
+                .Statistics(out stats)
+                .Where(e => e.Period == _period);
+                //.Customize(x => x.WaitForNonStaleResultsAsOfLastWrite());
```
Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluatedEmployees.cs]: You can remove if not needed.

**Comment by AlphaGit** [comment]
:+1:

File: source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluatedEmployees.cs (hunk 1 above)
Comment (Gkolocsar) [inline source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluatedEmployees.cs]: Will do, thanks!

**Pull request merged**